src/Store.js

```diff
@@ -163,7 +163,7 @@ class Store {
 
   async close () {
     if (this.options.onClose) {
-      await this.options.onClose(this.address.toString())
+      await this.options.onClose(this)
```

Comment:
Why not send just the address?

Reply:
The other `on____` functions require `this` so we can easily do `db._cache.open` in `orbit-db` I changed it just to be consistent. I don't think `onClose` is currently used anywhere else.

Reply:
:+1: what do you think of sending just the address in the other functions as well and retrieving the cache the same way we do in `onClose`?

Reply:
I guess it doesn't make too much of a difference since we can get what we need from OrbitDB.stores - should be equivalent objects. Just a bit more code on the OrbitDB side of things.